Content.Server/Atmos/Miasma/MiasmaSystem.cs

```diff
@@ -188,8 +188,16 @@ private void OnGibbed(EntityUid uid, PerishableComponent component, BeingGibbedE
 
         private void OnExamined(EntityUid uid, PerishableComponent component, ExaminedEvent args)
         {
-            if (component.Rotting)
+            if (!component.Rotting)
+                return;
+
+            // can't be a switch because these aren't constants
```

Comment:
Pretty sure it's possible with a switch expression, but it doesn't really matter and this comment doesn't really need to be in the codebase.